crates/vim9-gen/src/lib.rs

```diff
@@ -741,6 +741,12 @@ impl Generate for EchoCommand {
     }
 }
 
+impl Generate for ExecuteCommand {
+    fn gen(&self, state: &mut State) -> String {
+        format!("vim.fn.execute({})", self.expr.gen(state))
```

Comment:
I think there is an API command (maybe vim.api.nvim_exec, don't remember offhand) that we might want to use instead (in general, I prefer using API)

Reply:
`vim.api.nvim_command` seems to be the perfect option, so shifting to that :)